### PR TITLE
restyled 404 page (bug 972495)

### PR DESCRIPTION
**Pull request by spasovski**
We should probably do this for all error pages including fragment/app not found. The bug itself calls for a red background which was nixed at some point and a triangular exclamation icon which I didn't have an SVG of. For reference: https://bug972495.bugzilla.mozilla.org/attachment.cgi?id=8486754

### CODE REVIEW

File: src/media/img/pretty/alert_triangle.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
+        <g id="Triangle-1-+-Rectangle-1-+-Oval-1" sketch:type="MSLayerGroup">
+            <path d="M4.88354527,46.8062912 C1.08299827,53.5386888 4.26680686,59 11.9973049,59 L54.0026951,59 C61.7263632,59 64.9181318,53.5406906 61.1164547,46.8062912 L39.8835453,9.19370877 C36.0829983,2.46131121 29.9181318,2.45930937 26.1164547,9.19370877 L4.88354527,46.8062912 Z" id="Triangle-1" fill="#FF4E65" sketch:type="MSShapeGroup"></path>
+            <path d="M37,14 L29,14 L30.9928385,41 L35.0071615,41" id="Rectangle-1" fill="#FFFFFF" sketch:type="MSShapeGroup"></path>
+            <path d="M29,49 C29,46.790861 30.790861,45 33,45 C35.209139,45 37,46.790861 37,49 C37,51.209139 35.209139,53 33,53 C30.790861,53 29,51.209139 29,49 Z" id="Oval-1" fill="#FFFFFF" sketch:type="MSShapeGroup"></path>
+        </g>
+    </g>
+</svg>
```
Comment (cvan) [inline src/media/img/pretty/alert_triangle.svg]: newline

File: src/media/css/site.styl [hunk 2]
```diff
@@ -155,6 +155,34 @@ noscript {
...
+        max-width: 70%;
+    }
+}
+.error-page-heading {
+    font-weight: 100;
+    margin: 0 0 15px;
+    position: relative;
+
+    &:before {
+        background: url(../img/pretty/alert_triangle.svg) no-repeat 0 0 / 60px auto;
```
Comment (cvan) [inline src/media/css/site.styl]: does autoprefixer fix this to do a separate `background-size` rule?

Comment (cvan) [inline src/media/css/site.styl]: couldn't you do `100% auto` instead of `60px auto`?

File: src/media/css/site.styl [hunk 3]
```diff
@@ -155,6 +155,34 @@ noscript {
...
+        content: "";
+        height: 60px;
+        left: 50%;
+        margin-left: -30px;
```
Comment (cvan) [inline src/media/css/site.styl]: you could do `transform: translate(-50%, -50%)` instead to centre things

Comment (spasovski) [inline src/media/css/site.styl]: Both feel just as hacky but the translate will get rid of the fixed image size so will change it.

File: src/media/css/site.styl (hunk 2 above)
Comment (spasovski) [inline src/media/css/site.styl]: You brought up a good point for a different reason - 'auto' gets no love from IE or Safari so I'll probably get rid of the background size to begin with - it was there for the previous iteration.

File: src/media/img/pretty/alert_triangle.svg (hunk 1 above)
Comment (spasovski) [inline src/media/img/pretty/alert_triangle.svg]: sketch amirite?

**Comment by ngokevin** [comment]
r+

File: src/templates/errors/404.html [hunk 4]
```diff
@@ -1,12 +1,6 @@
 <section class="main">
-  <div class="subheader">
-    <h1>404</h1>
+  <div class="error-page">
+    <h1 class="error-page-heading">{{ _('404 Error') }}</h1>
```
Comment (ngokevin) [inline src/templates/errors/404.html]: this could probably say Content Not Found

**Pull request merged**